mysql-test/suite/innodb/t/innodb_row_lock_time_ms.test

```diff
@@ -19,18 +19,18 @@ SELECT * FROM t FOR UPDATE;
 --connection default
 COMMIT;
 
-SELECT variable_value > 100 FROM information_schema.global_status
+SELECT CAST(variable_value AS INT) > 100 FROM information_schema.global_status
   WHERE LOWER(variable_name) = 'innodb_row_lock_time';
-SELECT variable_value > 100 FROM information_schema.global_status
+SELECT CAST(variable_value AS INT) > 100 FROM information_schema.global_status
   WHERE LOWER(variable_name) = 'innodb_row_lock_time_max';
-SELECT variable_value > 100 FROM information_schema.global_status
+SELECT CAST(variable_value AS INT) > 100 FROM information_schema.global_status
   WHERE LOWER(variable_name) = 'innodb_row_lock_time_avg';
 
-SELECT count_reset > 100 FROM INFORMATION_SCHEMA.INNODB_METRICS
+SELECT CAST(count_reset AS INT) > 100 FROM INFORMATION_SCHEMA.INNODB_METRICS
   WHERE NAME="lock_row_lock_time";
-SELECT count_reset > 100 FROM INFORMATION_SCHEMA.INNODB_METRICS
+SELECT CAST(count_reset AS INT) > 100 FROM INFORMATION_SCHEMA.INNODB_METRICS
   WHERE NAME="lock_row_lock_time_max";
-SELECT count_reset > 100 FROM INFORMATION_SCHEMA.INNODB_METRICS
+SELECT CAST(count_reset AS INT) > 100 FROM INFORMATION_SCHEMA.INNODB_METRICS
   WHERE NAME="lock_row_lock_time_avg";
```

Comment:
As far as I can tell, `count_reset` is declared as `SLonglong()` in `storage/innobase/handler/i_s.cc`. Yet, we have observed some failures where the output differs. I don’t think that the `CAST` will help here.